specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2020-01-01/machineLearningServices.json

```diff
@@ -1452,6 +1452,15 @@
           },
           "readOnly": true,
           "description": "The current deployment state of workspace resource. The provisioningState is to indicate states for resource provisioning."
+        },
+        "resourceCmkUri": {
```

Comment:
BYOK is standardizing to the following schema since many services are implementing it at the moment.  "identityClientId" may not be necessary for you since you only support SystemAssigned MSIs.  However, I suggest including it in case you ever add userAssignedIdentities in the future.

```
"properties": {
   "encryption": {
      "status": "enabled",
      "keyVaultProperties": {
        "keyIdentifier": "string",
        "identityClientId": "string"
      }
    }
}
```

Reply:
ok. so we should be replacing the resourceCmkUri with the "encryption" property as you mentioned. is that the right understanding?